buildSrc/src/main/kotlin/org/cqfn/save/buildutils/DockerStackConfiguration.kt

```diff
@@ -15,7 +15,7 @@ import java.io.File
 import java.nio.file.Files
 import java.nio.file.Paths
 
-const val MYSQL_STARTUP_DELAY_MILLIS = 10_000L
+const val MYSQL_STARTUP_DELAY_MILLIS = 30_000L
```

Comment:
The most proper way here would be to integrate docker-java-api or some gradle plugin that wraps it. This way we could query docker API and check if database has started. But as a workaround it would do.

Reply:
Let's leave it as is for now because this pr is wanted to be merged asap in order to do another urgent tasks